pkgs/servers/home-assistant/component-packages.nix

```
@@ -94,7 +94,7 @@
     "bloomsky" = ps: with ps; [ ];
     "blueprint" = ps: with ps; [ ];
     "bluesound" = ps: with ps; [ xmltodict ];
-    "bluetooth_le_tracker" = ps: with ps; [ pygatt ];
+    "bluetooth_le_tracker" = ps: with ps; [ pygatt pygatt.extras-require.GATTTOOL ];
```

```suggestion
    "bluetooth_le_tracker" = ps: with ps; [ pygatt ] ++ pygatt.extras-require.GATTTOOL;
```
would be nice but I guess this is fine.

---

Oh yeah, they're nested lists now. Shoot.

---

Okay, I kinda went for a full reformat. PTAL.

---

Merging in a bit so I can continue updating home-assistant (two minor releases) based on these changes.